x-pack/plugins/security_solution/public/management/components/management_empty_state.tsx

```diff
@@ -0,0 +1,273 @@
+              onChange={handleSelectableOnChange}
+              data-test-subj="onboardingPolicySelect"
+            >
+              {(list) => {
+                return loading ? (
+                  <EuiSelectableMessage>
+                    <FormattedMessage
+                      id="xpack.securitySolution.endpoint.endpointList.loadingPolicies"
+                      defaultMessage="Loading policy configs"
+                    />
+                  </EuiSelectableMessage>
+                ) : selectionOptions.length ? (
+                  list
+                ) : (
+                  <FormattedMessage
+                    id="xpack.securitySolution.endpoint.endpointList.noPolicies"
+                    defaultMessage="There are no policies."
+                  />
+                );
+              }}
+            </EuiSelectable>
+          </>
+        ),
+      },
+      {
+        title: i18n.translate('xpack.securitySolution.endpoint.endpointList.stepTwoTitle', {
+          defaultMessage:
+            'Head over to Ingest to deploy your Agent with Endpoint Security enabled.',
+        }),
+        children: (
+          <EuiText color="subdued" size="xs">
+            <FormattedMessage
+              id="xpack.securitySolution.endpoint.endpointList.stepTwo"
+              defaultMessage="You'll be given a command in Ingest to get you started."
+            />
+          </EuiText>
+        ),
+      },
+    ],
+    [selectionOptions, handleSelectableOnChange, loading]
+  );
+
+  return (
+    <ManagementEmptyState
+      loading={loading}
+      onActionClick={onActionClick}
+      actionDisabled={actionDisabled}
+      dataTestSubj="emptyEndpointsTable"
+      steps={policySteps}
+      headerComponent={
+        <FormattedMessage
+          id="xpack.securitySolution.endpoint.endpointList.noEndpointsPrompt"
+          defaultMessage="You have a policy, but no Endpoints are deployed!"
+        />
+      }
+      bodyComponent={
+        <FormattedMessage
+          id="xpack.securitySolution.endpoint.endpointList.noEndpointsInstructions"
+          defaultMessage="Elastic Endpoint Security gives you the power to keep your endpoints safe from attack, as well as unparalleled visibility into any threat in your environment."
+        />
+      }
+    />
+  );
+});
+
+const ManagementEmptyState = React.memo<{
+  loading: boolean;
+  onActionClick?: (event: MouseEvent<HTMLAnchorElement | HTMLButtonElement>) => void;
+  actionDisabled?: boolean;
+  actionButton?: JSX.Element;
+  dataTestSubj: string;
+  steps?: ManagementStep[];
+  headerComponent: JSX.Element;
+  bodyComponent: JSX.Element;
+}>(
+  ({
+    loading,
+    onActionClick,
+    actionDisabled,
+    dataTestSubj,
+    steps,
+    actionButton,
+    headerComponent,
+    bodyComponent,
+  }) => {
+    return (
+      <div data-test-subj={dataTestSubj}>
+        {loading ? (
+          <EuiProgress size="xs" color="accent" className="essentialAnimation" />
```

Comment:
I take back my prior comment about using the Progress bar - I thought you had content inside of it, so that it would just show a small animation a top of the content that scrolled across the width of the area (similar to the animation on tables while loading)

For this use case, where all the user sees is a "busy animation", I would suggest going with the Loading component